qemu/tests/boot_e1000e_with_cpu_flag.py

```diff
@@ -22,7 +23,8 @@ def run(test, params, env):
     session_serial = vm.wait_for_serial_login(timeout=login_timeout)
     bcdedit_debug = params["bcdedit_debug"]
     bcdedit_cmd = params["bcdedit_cmd"]
-    ext_host = params.get("ext_host")
+    get_getway_cmd = params.get("get_getway_cmd")
+    ext_host = process.getoutput(get_getway_cmd, shell=True)
```

Comment:
```suggestion
    ext_host = utils_net.get_default_gateway()
```

Reply:
Thanks for your suggestion, I have changed this line. Please help me review it again.